4_params_munge.yml

```diff
@@ -37,12 +37,19 @@ targets:
       - 4_params_munge/out/mglp_depths.rds.ind
       - 4_params_munge/out/wbic_depths.rds.ind
       - 4_params_munge/out/ndgf_depths.rds.ind
+      - 4_params_munge/out/NY_manual_depths.rds.ind
 
 
   4_params_munge/out/toha_varying_kw.rds.ind:
     command: munge_daily_secchi(target_name,
       kw_files_zip_ind = '3_params_fetch/in/USGS_Kd_Predictions_byNHDHR.zip.ind')
 
+  # nhdhr_151957878: Pepacton Reservoir; nhdhr_120022743: Cannonsville Reservoir; nhdhr_149269381: Neversink Reservoir; nhdhr_61790779: Rondout Reservoir
+  4_params_munge/out/NY_manual_depths.rds.ind:
+    command: munge_simple_depth(target_name, nhdhr_151957878 = 55, nhdhr_120022743 = 42.672, nhdhr_149269381 = 53, nhdhr_61790779 = 53.3)
+  4_params_munge/out/NY_manual_depths.rds:
+    command: gd_get('4_params_munge/out/NY_manual_depths.rds.ind')
```

Comment:
In other repositories we've switched over to a system of putting all `gd_get`/`s3_get` calls in a separate `getters.yml`, which substantially cuts down on unnecessary rebuilds. we should probably do this in lake-temperature-model-prep too.